po/common/en_AU.po

```diff
@@ -4462,11 +4462,11 @@ msgid "Not a fair trade product"
 msgstr ""
 
 msgctxt "attribute_labels_fair_trade_yes_description_short"
-msgid "Helps producers in developping countries."
+msgid "Helps producers in developing countries."
```

Comment:
Github's code search didn't reveal other occurences of this id, but there may be more that need tweaking.